### DISCUSSION TITLE
BOM missing?

### DISCUSSION THREAD

**Discussion by olDirdey** [sanni/cartreader#958] [Hardware]
### OSCR firmware version

asde

### OSCR hardware version

HW1

### Attach OSCR_LOG.txt file here

asd

### System used

Game Boy (Color)

### Describe the issue.

where is the BOM for the project?

**Comment by Ancyker** [comment]
https://github.com/sanni/cartreader/wiki/What-to-order

**Comment by olDirdey** [comment]
I don't see any bom there either, just a much too small picture of an aliexpress shopping cart

**Comment by Ancyker** [comment]
Scroll down, there are links to everything.

**Comment by olDirdey** [comment]
isn't there just a part number so that I can search the parts from a proper source? I don´t like to order stuff from ali

**Comment by Ancyker** [comment]
For most things? No. For some things? Yes.

* Get the [Clock Generator from Adafruit](https://www.adafruit.com/product/2045) (or [here](https://store.starshade.dev/product/oscr-and-accessories/oscr-accessories/rtc-clkgen-module)).
* Bulk Capacitor: [PLS1A471MDO1TD](https://www.digikey.com/en/products/detail/nichicon/PLS1A471MDO1TD/4991343)
* 1K Resistor: [PR01000101001JR500](https://www.digikey.com/en/products/detail/vishay-beyschlag-draloric-bc-components/PR01000101001JR500/595993)
* CIC: [PIC12F629-I/SN](https://www.digikey.com/en/products/detail/microchip-technology/PIC12F629-I-SN/459170) (or [here](https://store.starshade.dev/product/oscr-hw5-cic))

The rest either isn't available from a reputable source or is so simple it's not worth the effort (i.e. pin headers).

**Comment by olDirdey** [comment]
thanks for your help

**Comment by skaterdude** [comment]
I kinda ran into the same problem.  So I started putting together a worksheet with all the info, since everything looks like it needs a worksheet to me, LoL.
https://docs.google.com/spreadsheets/d/134r0eYbTgUWuzSZPGuHhNnfH5RoNqBjJHmNBlyzatnQ/edit?usp=sharing

Happy to take suggestions as to how to improve it, or updated/alternate sources and prices.

**Reply by sanni** [reply]
If you want you can add the spreadsheet to the wiki page: https://github.com/sanni/cartreader/wiki/What-to-order
That way it's easier to find and I'm sure others will be thankful. 😃

**Reply by skaterdude** [reply]
At the moment it's still a little in flux.  I'm waiting for my parts orders to arrive in the next week.  Once I'm sure everything fits and works, that would be a good idea.  Can I edit the wiki like a conventional wiki, or do I have to do a pull in github?  I'm a little rusty at that process, but I'm pretty familiar with editing wikis.

**Reply by sanni** [reply]
You can just click the edit button and change the text directly in the form, no pull request needed.

**Reply by skaterdude** [reply]
Cool.  I'm going to wait until I verify the build before I post it to the What-To-Order page.  Already found that cannibalized N64 extension cables do not work with either of the 3D printed cases I tried out.  Have not received the bare connectors yet.  In the meantime, I'll continue to update the Google Sheet.

**Reply by breyell** [reply]
> Cool. I'm going to wait until I verify the build before I post it to the What-To-Order page. Already found that cannibalized N64 extension cables do not work with either of the 3D printed cases I tried out. Have not received the bare connectors yet. In the meantime, I'll continue to update the Google Sheet.

If they're the extensions like in [this video](https://youtu.be/yZaGvuH10MY?si=9RY7JNZgAdbI5Uik&t=1423), you can crack the outer portion and use them.

**Reply by skaterdude** [reply]
Excellent.  Thanks!  I think the sockets alone are cheaper, but using the extension gets you all the wires ready to go.
